SU-SWS/ood-giving-campaign
# GIVCAMP-269 GIVCAMP-206: Loading, Cache, and 404.

# READY FOR REVIEW

# Summary
- Trying out new cache controls
- Added loading page
- New settings don't seem to be showing the loading page after all

# Steps to test

1. View https://deploy-preview-180--giving-campaign.netlify.app/ in Safari
2. View loading page https://deploy-preview-180--giving-campaign.netlify.app/loading to preview what the loading will look like
3. Navigate back to the home page by clicking the logo
4. Navigate to a new page using an in page link
5. Click the back button
6. Wait for page to load and scroll around
7. See no back button issue
8. Navigate to  https://deploy-preview-180--giving-campaign.netlify.app/this-page-does-not-exist
9. See 404 page
10. Log in to Netlify and view the [function logs](https://app.netlify.com/sites/giving-campaign/logs/functions/?scope=deploypreview:180)
11. Open up the SSR and ISR function logs
12. Set the time to 'last hour'
13. Wait and you shouldn't see any logs related to pages

For bonus points, make an edit on Storyblok and then 'rebuild' the Netlify environment then check to see if the content shows up.
